Mathlib/Data/Fin/Basic.lean

```diff
@@ -251,17 +246,29 @@ This one instead uses a `NeZero n` typeclass hypothesis.
 protected theorem zero_le' [NeZero n] (a : Fin n) : 0 ≤ a :=
   Nat.zero_le a.val
 
+@[simp, norm_cast]
```

Comment:
I had to disable this `@[simp]` in the `Imo2024Q5` file, so not sure if it should be a `@[simp]` lemma. CC: @kim-em

Reply:
I think it's okay to have this be a simp lemma. The places you had to disable it were using `Fin.ext_iff` *inside* a (even nonterminal in two cases) `simp` call, and arguably that lemma doesn't really belong inside `simp`, so I think it's reasonable.